test/user/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2.4'
+
+services:
+  alpine:
+    image: alpine:3.17.3
+    tty: true 
+    volumes:
+      - app_data:/tmp
+    labels:
+      - docker-volume-backup.archive-pre.user=testuser
+      - docker-volume-backup.archive-pre=/bin/sh -c 'whoami'
```

Comment:
Wouldn't this still need to store the output somewhere, i.e.

```
docker-volume-backup.archive-pre=/bin/sh -c 'whoami > /tmp/whoami.txt'
```

Reply:
Done